src/user-guide/writing-workflows/external-triggers.rst

```diff
@@ -363,6 +372,45 @@ An example xrandom trigger workflow:
 .. literalinclude:: ../../workflows/xtrigger/xrandom/flow.cylc
    :language: cylc
 
+.. _xrandom.validate:
+
+Validation example using xrandom
+""""""""""""""""""""""""""""""""
+
+The xrandom xtrigger module might also contain a ``validate`` function.
+
+This will be run on the inputs to an xtrigger, and should raise
+a ``WorkflowConfigError`` with a meaningful description if a condition
```

Comment:
```suggestion
The ``xrandom`` xtrigger module contains a ``validate`` function.

This will be run on the inputs to the xtrigger when calling 
``cylc validate`` or before the workflow starts, and should raise a
:py:exc:`cylc.flow.exceptions.WorkflowConfigError` 
with a meaningful description if a condition
```